src/storage/vpid_utilities.hpp

```diff
@@ -19,6 +19,9 @@
 #ifndef _VPID_UTILS_HPP_
 #define _VPID_UTILS_HPP_
 
+#include "dbtype_def.h"
+
+#include <algorithm>
 #include <cstdint>
 #include <cstdio>
```

Comment:
I think \<functional\> has to be included.

Reply:
done